app/views/shared/_sidenav.html.erb

```diff
@@ -1,38 +0,0 @@
-<aside class="usa-width-one-fourth usa-layout-docs-sidenav">
-  <ul class="usa-sidenav-list">
-    <li>
-      <%= active_link_to 'Moving with the Military', page_path('moving-guide'), class_active: 'usa-current' %>
```

Comment:
Given the removals here, I _think_ you could now remove the `active_link_to` gem. Double-check the rest of the codebase and make sure it isn't being used anywhere. If not, remove it from `Gemfile` and re-run `bundle install` to get it out of `Gemfile.lock`.

Reply:
You're right! It's not used anymore. What does active_link_to do? I'm wondering if we'll need it in the future since I spoke to @lfantano and she said we *might* reintroduce the sidenav for certain things, e.g. switching the service-specific info tabs to a side nav, although it's currently unclear if we will. I can remove it and just add it back if we end up needing it again.

Reply:
[active_link_to](https://github.com/comfy/active_link_to) is a gem that adds a view helper (`active_link_to`) that accepts arguments similar to `link_to` but with a little bit of clever logic to tack on "active" class names of your choosing for the purpose of appropriately styling a navigation item for the page a user is currently viewing.

It's a pretty useful gem, but if we're no longer using it, I recommend removing it from the Gemfile and taking it out of the `LICENSE.md` file. That removal can (and maybe should) be a separate PR.

Reply:
Ok, will remove in separate PR